tdrs-frontend/src/components/Routes/Routes.js

```diff
@@ -20,10 +20,10 @@ const Routes = () => {
       <Route exact path="/login">
         <LoginCallback />
       </Route>
-      <PrivateRoute exact path="/edit-profile">
+      <PrivateRoute exact title="Request Access" path="/edit-profile">
         <EditProfile />
       </PrivateRoute>
-      <PrivateRoute exact path="/request">
+      <PrivateRoute exact title="Request Submitted" path="/request">
```

Comment:
The titles that were previously in the h1 tag of each of these pages is now passed in via title prop to the router.